accord-core/src/main/java/accord/impl/CommandChange.java

```diff
@@ -90,18 +91,18 @@ public enum Field
     {
         PARTICIPANTS, // stored first so we can index it
         SAVE_STATUS,
-        PARTIAL_DEPS,
```

Comment:
is there a reason to shuffle this around?  confirmed nothing was lost, but journal does rely on `field.ordinal()` which makes this a breaking change; so would like to know why we should reorder the fields

Reply:
To make the field order consistent throughout this class and also broadly consistent with the rest of the codebase (except where otherwise necessary for efficiency, ie participants)

Nothing has been released yet, so this is not a breaking change, which is why it is important to make this kind of consistency change before that happens